### PR TITLE
Update operator version to reflect kubeflow version

### PR DESCRIPTION
**Pull request by adrian555**
Towards #284 

This PR adds new bundled package files for the new version of kubeflow operator. This allows the users to install the correct version of kubeflow with the operator.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kfctl/285)

### CODE REVIEW

File: cmd/manager/main.go [hunk 1]
```diff
@@ -25,23 +25,28 @@ import (
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/util/intstr"
 	"sigs.k8s.io/controller-runtime/pkg/client/config"
-	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	log "github.com/sirupsen/logrus"
```
Comment (animeshsingh) [inline cmd/manager/main.go]: Is this version approved to be used? Looks like someone's individual github repo?

File: deploy/olm-catalog/kubeflow/kubeflow.package.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 channels:
-- currentCSV: kubeflow.v0.1.0
+- currentCSV: kubeflow.v1.0.1
```
Comment (animeshsingh) [inline deploy/olm-catalog/kubeflow/kubeflow.package.yaml]: What will correspond to version 1.0?

Comment (adrian555) [reply deploy/olm-catalog/kubeflow/kubeflow.package.yaml]: thinking whether we should update v1.0-branch. If it is not possible to update that, then we will have to spell out in the document.

File: cmd/manager/main.go (hunk 1 above)
Comment (adrian555) [reply cmd/manager/main.go]: `github.com/sirupsen/logrus`? that is what almost all other packages in `kfctl`is using.

**Comment by adrian555** [comment]
> Does any thing change in operator readme to regenerate these files if someone needs to modify?

the current `Build Instructions`  section https://github.com/kubeflow/kfctl/blob/master/operator.md#build-instructions is still good.

File: deploy/olm-catalog/kubeflow/kubeflow.package.yaml (hunk 2 above)
Comment (animeshsingh) [reply deploy/olm-catalog/kubeflow/kubeflow.package.yaml]: Lets change the version to 1.0.0 

otherwise
/lgtm

Comment (adrian555) [reply deploy/olm-catalog/kubeflow/kubeflow.package.yaml]: Done. Thanks @animeshsingh

**Pull request merged**